### ISSUE TITLE
[3.3.5a] A bunch of mobs spam "calling for help" at low hp

### ISSUE THREAD

**Issue by Treeston** [TrinityCore/TrinityCore#15526]
3.3.5 branch, Rev b70fac179edf+, TDB 335.59.

A bunch of mobs have a smartscript event to run away (or run for assist) at low health without no-repeat being set, causing the emote to be spammed very rapidly.

```sql
update `smart_scripts` set `event_flags`=(`event_flags`|1) where
`entryorguid` in (619,2287,2719,3987,10817,17210) and
`source_type` = 0 and
`action_type` in (25,39)
```

**Comment by tkrokli** [comment]
Thanks. Malcrom posted some SQL in your closed PR, #15525, any idea if it is related?

BTW, missing semicolon at the end of the SQL and parenthesis around `event_flags` is redundant.
Suggestion for how it would look when posted by the TC devs:
```sql
UPDATE `smart_scripts` SET `event_flags` = `event_flags` | 1 WHERE `entryorguid` IN (619,2287,2719,3987,10817,17210) AND `source_type` = 0 AND `action_type` IN (25,39);

**Comment by Treeston** [comment]
I seem to remember there being some incorrectly assigned comments when I fixed them a while ago in our server DB, yeah. He probably fixed those.
